data/yam/agama/auto/post_script.jsonnet

```diff
@@ -0,0 +1,14 @@
+{
+  "scripts": {
```

Comment:
jsonnet doesn't need double quotes, please check the documentation.

Reply:
Ok, I will update this in next PR.